components/renku_data_services/storage/rclone.py

```diff
@@ -369,6 +377,8 @@ def validate_config(
             raise errors.ValidationError(message=f"The following fields are required but missing:\n{missing_str}")
 
         for key in keys:
+            self.check_unsafe_option(key)
```

Comment:
Note: Code below (L390) basically allows any option to be provided by hand (advanced config). If some other options are unsafe, they need to be rejected by `check_unsafe_option()`. This means that the OAuth2 options we hide can in fact be used, though I'm not sure what would happen then. At least, those should not execute harmful code paths.